src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -87,130 +96,208 @@
 
    private static final Logger LOG = LoggerFactory.getLogger(ReportController.class);
 
-   private DatePicker datePicker; // for calendar element
-
    private Model model;
 
    private Controller controller;
 
+   private Stage stage;
+
    private ColorTimeLine colorTimeLine;
 
+   private LocalDate currentReportDate;
+
    @FXML
    private void initialize() {
       LOG.info("Init reportController");
+      currentReportDate = LocalDate.now();
+      initTableView();
+      colorTimeLine = new ColorTimeLine(colorTimeLineCanvas);
+   }
 
-      this.datePicker = new DatePicker(LocalDate.now());
-      this.datePicker.valueProperty().addListener((observable, oldvalue, newvalue) -> {
-         LOG.info("Datepicker selected value changed to {}", newvalue);
-         updateReport(newvalue);
-      });
+   private void initTableView() {
+      final TreeTableColumn<TableRow, String> noteColumn = new TreeTableColumn<>("Notes");
+      noteColumn.setCellValueFactory(new TreeItemPropertyValueFactory<TableRow, String>("notes"));
+      noteColumn.setMinWidth(200);
+      this.workTableTreeView.getColumns().add(noteColumn);
+
+      final TreeTableColumn<TableRow, String> timeRangeColumn = new TreeTableColumn<>("Timeslot");
+      timeRangeColumn.setCellValueFactory(new TreeItemPropertyValueFactory<TableRow, String>("timeRange"));
+      timeRangeColumn.setMinWidth(100);
```

Comment:
it is always cutting the timeslot strings. please make it wider. We may also make the stage a little wider if need be

Reply:
--  Duration is always to big :) so we can make duration smaller